Integrations/integration-Guidance_Encase_Endpoint.yml

```diff
@@ -0,0 +1,243 @@
Y7dmk+k9hqpBoCv873Y7OU832vtbOg0InD6PAdHuecDismZi6fVehs0lCAoCgDoDYFRAAAAAAAbkrPzjGKRTbX8Mrz0Yp/DdOzJNuxunBfJr3uYWJQw73A1DNf8/XPvyieZyygAGxGBI+sGNZV47z7qGDkDwkX19EyFk2r9LIczGQKXIrtmejhY5xqQM07V4WLNtj6qBMB1yUHqS6fpvSnTF8C3edN5vs7GffqmgqPief9M5/HOfR9Q6B6krs+BH0tQFADQK/9RBQAAAAAAmxWLpNPDIJVPNRx+skVV+XnN39/v+Otf9/wFRgFdunaeFxj3112I9tRsQadrPOdugeqb6EHF6uJtKn/WVNZt670ag7aox6ucFQ6Af8754nq26QzHbyNjaSrPtArwVLGBVXr4oxIUBQBQnMAoAAAAAIAWiMwNqcSOoa8LH3q0RQt31g3sGXX1hac2jqCodRdVLLwTgY453fDzT574e9Nqc4t7z7Yhk+SK185BtX7WsbYba+laPE/vox9ulvRnv5Ua1/K8DoCsRcFRL1I5N04DT7j3eJbKvFpurLBpgqIAgF4SGAUAAAAA0CKz929ikffzqtyCnwiWGW1J9a0bGLXT4QVO4xbUH0Djl81qswFG8yde6y/zuW/CqW5T9NrZdocyWzQ2hz+q1s9idz13l5EE4N/jbFuCoyKj419pnD7SKsAqchbXRSoHLTgdQVEAQG8JjAIAAAAAaJm80HpYlVvwM9ySqisR2DPu6GsfFep3AF26Xm4ywGiy4d9/iotUZzM9Z7lje3rYlgXNFm43Ox8rMX/f3eDYBtDmud91cNRFC07n1zSfEMgKPHTfEfd9f1brZ3lfV8xRBUUBAL0mMAoAAAAAoIVuLPgpYbgldTYvcJhx11533nl2d83DnHnXAR21iQxIn9e95qTfX2xg7JUt6p/X+50teq00MxddFKzvgzTH854F+PdYG5+VRKbnzy04nRepnHc48zRQkzQuDFKJe8a3LTidCIoaCooCAPpOYBQAAAAAQEvlBT+vCxxqd4t2Mf605u/vpLoad+w1lzhfWQmALl8rm14YWypYYdrgOV81/Hxtt01ZlHY7OLfp8pgUc6p3hQ73StsB3DnWRtbQYdWO4KjYpOSvnBUGIIKiIoto3KcetOB0roOizrUMANB3AqMAAAAAAFps9v5NLL4usdhnW3Ywnhc4xqQrLzZ2oE0Phy2pN4BNaTKrykWpnbbzcS6amlLkRcRbLwea7G7Zyx5r+Ubn7zGXLJUR7lQmEoA7x9rr4KiPLTmlt5EdJt+jA9t5n/EsZ/z8o2pHdtr4PHlfUBQAsC0ERgEAAAAAtF+JBd/bkjGqROajLmVWmBQ4xoVFEkAPxv6rDo27N007et5dNt7C13wguKZxo0LjUiyqnW1R9leA1SeA799cphLX9bYER0V2mPM0Zh9pHdiyid/xybBaZol61ZJTiqCoyBS10DoAwLYQGAUAAAAA0HKFMlPsb0ldLaoyO/RP2r4ANS+6KJEtauZdBnR87L9saCwrli3qhiayXZ1ZEPePa+fBlr58i7SbH5dGhQ63a74GcO+YO04Pr1tyOhHQ+muacwhqhe25x5ikhz+r9mSljWDRoYzBAMC2ERgFAAAAANANZ6pgZdMCx4jFDJOWv87Tlh0HYJOaGLOLj5d5sdrHrp13h21zcNDIAu1mpff3PD28K3S4yPrlvQzw/TE3xsiXLTqlF6ks0tg90jrQ08n18cl+KpEl6m2LTutjBIsKigIAtpHAKAAAAAAA+iZ21L8qcJxXObNE6+SFsXsFDiWLCNCPgb9cxsDvievKtKZj1xnsEFmuZJpZXjsH1XKR8raKDBZjPaHxsWlScGyKuak2BPj+mBtztZ8LfR5Q6tr7h+xR0Mt7i5jj/VWV+WyulJc5gx4AwFYSGAUAAAAAcI+8++dQTXRH3hV1WupweSF1m/pk7Dj9qtDhZB4A+mRa47FP69p1Ox03dhmvK6jLOP/NpMAxzjZYPhc4/yPdYDPTt6rcIv3TuD9RpQD3zqv2C103S7nOHjXWQtDxSd3xybCFWaJinvk8B4cCAGytH1UBAAAAAMC9YlffP0fHJy99wdwpsRC8RPBQ7PAcwVHDuhbEP0ZeCFuqH8oiAvRKXKdzRr2dwoe+quoPMIqx/aCG8zZ3WV4/Yz53uOZhPm5yB/b8GhZr9u/dCLB2/W98bLrMge1/Fpyb7rdhbgrQ0nF3kTe4mdUwv1pn/P6Qg6PGMjdDJ+8nJlW5jYpK+ZzHlHOtBABsOxmjAAAAAABWEwtYpvmL8E2wM/wj5EVGHwsdbi+V+Qbb/oscFDWvyi34n+gpQA9N6zhm3QEIOfj6qvRhBU58ddTSvvWYPhJtWSKgaaw7bKT9Yg73utDhdgv1BYA+j7uXqQzTj7+17NQiUOs83d+7H4eOyAHuEXjUtqCoyCo7FBQFALAkMAoAAAAA4H43FxRHpoF5DlBpTN7peN1gmG38knxS8FgbDY6qISjqswxoQE+dduSYTTzPRHf4at3AqIsc2LJpJa7dL9K8YqBLNC/1oXiPfyp0uIOcIQ+A+8femAO8bNlpxX392zSOX2e2Aloo5sypxD3AH9UyML1NfovgTxthAAB8IzAKAAAAAOAed+y6GQEyfzW8u2+JLAeXW9h2i/TwruAho+3PNxAYN67KBkWV6lMAbR37zwoe8mM+ZhOmBY911uB5t1q+jq57DT1tSf+O+cCFeUCnjQu1YXiV+zcA918/Y471c1U+O+e6ItDizw1nJwf+ff/wLH/uG58JH7TwFF/moE8AAG4QGAUAAAAA8DSxu+953bv75uO/WPc4LclysAmxkPmi4PFi4VIjgXF5IcYs/fihKhsU9WmL+wOwHaYFjzVp6qRzINPHgtc/yrXhrGVzm3WNLcDejLyr/6hkf2g6aB+go+NvBDgMqrIB9KVEdvJFwxvwAHdI78OYp8V48bYq+1lcCfH55s8ywAMA3E1gFAAAAADAw
```

Comment:
It's shorter and nicer to do the following:
1. Build a bodyDictionary with the command as key and the xml body as value
2. Inside the switch, take the body according to the command
3. Replace it with args
4. Send the request with the modified xml
5. Get the task.message